landingai/pipeline/frameset.py

```diff
@@ -243,22 +332,35 @@ def downsize(
                 frame.image = frame.image.resize((width, height))
         return self
 
```

Comment:
Can we try to implement functions on a single place? For example in this case we should rely on the `Frame.downsize` implementation

Reply:
Great. I think you are also missing `show_image`

Reply:
Good catch. Let me add it.